students/jesse_miller/session03/mailroom.py

```diff
@@ -86,61 +110,81 @@ def donor_list():
     print("-" * 15)
     print("List of donors: ")
     print("-" * 15)
-    for donor in DONORS:
+    for donor in donors:
         print(donor)
     print("")
-    #mail_menu()
 
 def donor_mail():
     """
     This section allows the user to mail a donor
     """
+    global Current_Donor
```

Comment:
not sure why this is `global` as well as you don't want to capitalize the first letters